modules/navier_stokes/src/fvkernels/INSFVMomentumAdvection.C

```diff
@@ -65,6 +94,18 @@ INSFVMomentumAdvection::initialSetup()
       std::set<ExecFlagType>({EXEC_ALWAYS}),
       _mesh,
       this->blockIDs());
+
+  if (_use_norm_for_momentum_limiter)
+    _mom_abs = std::make_unique<PiecewiseByBlockLambdaFunctor<ADReal>>(
+        "rho_vel",
+        [this](const auto & r, const auto & t) -> ADReal
+        {
+          return _rho(r, t) * NS::computeSpeed(ADRealVectorValue(_u(r, t), _v(r, t), _w(r, t))) /
+                 epsilon()(r, t);
```

Comment:
If we solve for the superficial velocity, why limit using the interstitial one?

Reply:
the limiters are computed from the interstitial momentum currently, so I did not change that